src/users/UserList.js

```diff
@@ -47,7 +47,6 @@ export default function UserList(props: Props) {
       keyboardShouldPersistTaps="always"
       initialNumToRender={20}
       sections={sections}
-      keyExtractor={item => item}
```

Comment:
```
UserList [nfc]: Remove keyExtractor.

This was redundant with the `key` prop on the UserItems.
```

`key` isn't quite correctly called a prop; I tend to call it the "`key` attribute". You can't access it on `props`, and of course it has its special role that props don't have (https://reactjs.org/docs/lists-and-keys.html, https://reactjs.org/docs/reconciliation.html#keys).